examples/code_review_agent/skills/code-review/scripts/check_secrets.sh

```
@@ -0,0 +1,59 @@
+    "password"
+    "credential"
+    "secret"
+)
+
+echo "Scanning for potential secrets in: $FILE_PATH" >> out/secrets_scan.txt
+echo "" >> out/secrets_scan.txt
+
+for pattern in "${SECRET_PATTERNS[@]}"; do
+    matches=$(grep -i -n "$pattern" "$FILE_PATH" | head -20)
+    if [ -n "$matches" ]; then
+        echo "Found potential '$pattern' references:" >> out/secrets_scan.txt
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)